cli_ftd/main.c

```diff
@@ -120,6 +122,8 @@ int main(void) {
 
   cli_taskCreate();
 
+  udp_comm_taskCreate();
```

Comment:
Isnt this also CC1352R1 specific ?

Reply:
No, this is platform-independent.

Reply:
I will modify CMakeLists.txt to include it as common platform target.